dashboard/modules/stats_collector/stats_collector_head.py

```diff
@@ -220,13 +220,23 @@ def _process_actor_table_data(data):
         # Receive actors from channel.
         async for sender, msg in receiver.iter():
             try:
-                _, actor_table_data = msg
+                actor_id, actor_table_data = msg
                 pubsub_message = ray.gcs_utils.PubSubMessage.FromString(
                     actor_table_data)
                 message = ray.gcs_utils.ActorTableData.FromString(
                     pubsub_message.data)
                 actor_table_data = actor_table_data_to_dict(message)
                 _process_actor_table_data(actor_table_data)
+                # If actor is not new registered but udpated, we only update
+                # states related fields.
+                if actor_table_data["state"] != "DEPENDENCIES_UNREADY":
+                    actor_id = actor_id.decode("UTF-8")[len("ACTOR:"):]
```

Comment:
Can you use `gcs_utils.TablePrefix_ACTOR_string` instead of hardcoding it?

Reply:
no problem.